### PR TITLE
Add advice on alcohol at events

### PR DESCRIPTION
**Pull request by danielfone**
Sneaked in a change for the Rails Camp details too.

screen shot 2017-07-19 at 9 23 21 pm

### CODE REVIEW

File: events/alcohol-advice.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Event Alcohol Advice | Ruby New Zealand Incorporated</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,900" rel="stylesheet">
```
Comment (raquelxmoss) [inline events/alcohol-advice.html]: Could we please use the same font as the main page? It's [Poppins](https://fonts.google.com/specimen/Poppins)

File: events/alcohol-advice.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  <ul>
+    <li>Setting drinking expectations in advance with those attending</li>
+    <li>Keeping an eye on alcohol intake of guests</li>
+    <li>Providing food and water</li>
+    <li>Focusing activity on the social aspect of the occasion</li>
+  </ul>
+
+  <p>
+    If you have any questions about alcohol at Ruby New Zealand events,
+    please get in touch with the <a href="mailto:committee@ruby.nz">committee</a>. Thanks! ✨
```
Comment (raquelxmoss) [inline events/alcohol-advice.html]: 🤔 Would it be gauche to throw in a 🍻 emoji? Haha

**Review by raquelxmoss — CHANGES_REQUESTED** [review]
Nice! Please just change the font, then I am good for this to go out.

**Comment by danielfone** [comment]
@raquelxmoss I've fixed fonts and added gauche. Let me know if you want me to squash.

**Review by raquelxmoss — APPROVED** [review]

**Comment by raquelxmoss** [comment]
Looks good, merge and deploy whenever you like

**Pull request merged**